tools/sim/bridge.py

```diff
@@ -12,26 +12,19 @@
 import pyopencl as cl
 import pyopencl.array as cl_array
 
-from lib.can import can_function
-
+from selfdrive.manager.helpers import unblock_stdout
+from tools.sim.lib.can import can_function
 import cereal.messaging as messaging
 from cereal import log
-from cereal.visionipc.visionipc_pyx import VisionIpcServer, VisionStreamType  # pylint: disable=no-name-in-module, import-error
+from cereal.visionipc.visionipc_pyx import VisionIpcServer, \
+  VisionStreamType  # pylint: disable=no-name-in-module, import-error
```

Comment:
why a new line for this? and if using a new line, indent to the same level as `VisionIpcServer`